ros/api/v1/hosts.py

```diff
@@ -472,6 +482,27 @@ def get(self):
         io_states_dict['undersized'] = -1
         total_conditions = totals['cpu'] + totals['memory'] + totals['io']
 
+        current_utc_datetime = datetime.utcnow().replace(tzinfo=pytz.utc)
```

Comment:
```suggestion
       current_utc_datetime = datetime.now(timezone.utc)
```
Reference - https://docs.python.org/3/library/datetime.html#datetime.datetime.now

Reply:
The timezone metadata seems to be a bit different for each of the functions, I'm investigating.
I'll push shortly, thanks!
```
datetime.now(timezone.utc)
datetime.datetime(2022, 10, 14, 19, 12, 15, 782847, tzinfo=datetime.timezone.utc)
datetime.utcnow().replace(tzinfo=pytz.utc)
datetime.datetime(2022, 10, 14, 19, 12, 19, 228398, tzinfo=<UTC>)
```

Reply:
@kgaikwad Change pushed